cpp/arcticdb/storage/python_bindings.cpp

```diff
@@ -95,21 +97,77 @@ void register_bindings(py::module& storage, py::exception<arcticdb::ArcticExcept
 
     storage.def("create_library_index", &create_library_index);
 
-    storage.def("create_mem_config_resolver", [](const py::object & env_config_map_py) -> std::shared_ptr<ConfigResolver> {
+    
+    py::enum_<s3::AWSAuthMethod>(storage, "AWSAuthMethod")
+        .value("DISABLED", s3::AWSAuthMethod::DISABLED)
+        .value("DEFAULT_CREDENTIALS_PROVIDER_CHAIN", s3::AWSAuthMethod::DEFAULT_CREDENTIALS_PROVIDER_CHAIN)
+        .value("STS_PROFILE_CREDENTIALS_PROVIDER", s3::AWSAuthMethod::STS_PROFILE_CREDENTIALS_PROVIDER);
+
+    auto s3settings_def = py::class_<s3::S3Settings>(storage, "S3Settings")
```

Comment:
Why do we need the name `s3settings_def` for this? Can't we leave it anonymous like the others?